content/03.methods.md

```diff
@@ -23,22 +23,32 @@ MedRxiv mainly focuses on preprints that mention patient research, while bioRxiv
 We downloaded a snapshot of both resources on March 4th, 2022, using their respective Amazon S3 bucket [@https://www.biorxiv.org/tdm; @https://www.medrxiv.org/tdm].
 This snapshot contained 172,868 BioRxiv preprints and 37,517 MedRxiv preprints.
 These resources allow authors to post multiple versions of a single preprint.
-To prevent duplication bias, we filtered every preprint to its most recent version and sorted each preprint into its respective posted year.
+We filtered every preprint to its most recent version to prevent duplication bias and sorted each preprint into its respective posted year.
 Unlike Pubtator Central, these filtered preprints do not contain any annotations.
 Therefore, we used TaggerOne [@doi:10.1093/bioinformatics/btw343] to tag every chemical and disease entity and GNormplus [@doi:10.1155/2015/918710] to tag every gene and species entity for our preprint set.
 Once tagged, we used spacy to preprocess every preprint as described in our Pubtator Central section.
 
 ## Constructing Word Embeddings for Semantic Change Detection
 
+![
+A. The first step of our data pipeline is where PMCOA papers and BioRxiv/MedRxiv preprints are binned by their respective posting year.
```

Comment:
This figure is SO helpful! 👏

Reply:
FYI every once in a while a journal has yelled at me for putting a figure in the methods section. If that happens, we need to put this in the results and add one sentence there: "We developed a pipeline for ... (Figure X)."